CI Failure February 2024

CI on Linux/conda is failing with error:
~~~
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
libgl1 is already the newest version (1.4.0-1).
libgl1 set to manually installed.
libselinux1 is already the newest version (3.3-1build2).
libselinux1 set to manually installed.
libxdamage1 is already the newest version (1:1.1.5-2build2).
libxdamage1 set to manually installed.
libxext6 is already the newest version (2:1.3.4-1build1).
libxext6 set to manually installed.
libxxf86vm1 is already the newest version (1:1.1.4-1build3).
libxxf86vm1 set to manually installed.
libgl1-mesa-dri is already the newest version (23.0.4-0ubuntu1~22.04.1).
libgl1-mesa-dri set to manually installed.
The following additional packages will be installed:
  libegl-mesa0
The following NEW packages will be installed:
  libegl-mesa0 libegl1 libopengl0
0 upgraded, 3 newly installed, 0 to remove and 18 not upgraded.
Need to get 182 kB of archives.
After this operation, 709 kB of additional disk space will be used.
Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [142 B]
Ign:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 libegl-mesa0 amd64 23.0.4-0ubuntu1~22.04.1
Get:3 http://azure.archive.ubuntu.com/ubuntu jammy/main amd64 libegl1 amd64 1.4.0-1 [28.6 kB]
Ign:2 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libegl-mesa0 amd64 23.0.4-0ubuntu1~22.04.1
Get:4 http://azure.archive.ubuntu.com/ubuntu jammy/main amd64 libopengl0 amd64 1.4.0-1 [36.5 kB]
Err:2 http://security.ubuntu.com/ubuntu jammy-updates/main amd64 libegl-mesa0 amd64 23.0.4-0ubuntu1~22.04.1
  404  Not Found [IP: 52.252.75.106 80]
Fetched 65.0 kB in 0s (316 kB/s)
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/m/mesa/libegl-mesa0_23.0.4-0ubuntu1%7e22.04.1_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.
~~~

--- a/.github/workflows/conda-forge.yml
+++ b/.github/workflows/conda-forge.yml
@@ -31,13 +31,15 @@ jobs:
         environment-file: ci_env.yml
 
     # Additional dependencies useful only on Linux
-    - name: Additional Dependencies (Linux])
+    - name: Additional Dependencies (Linux)
       if: contains(matrix.os, 'ubuntu')
       shell: bash -l {0}
       run: |
         # Additional dependencies only useful on Linux
         # See https://github.com/robotology/robotology-superbuild/issues/477
         micromamba install expat-cos6-x86_64 freeglut libselinux-cos6-x86_64 libxau-cos6-x86_64 libxcb-cos6-x86_64 libxdamage-cos6-x86_64 libxext-cos6-x86_64 libxfixes-cos6-x86_64 libxxf86vm-cos6-x86_64 mesa-libgl-cos6-x86_64 mesa-libgl-devel-cos6-x86_64
+        # Update package lists
+        sudo apt-get update
         # See https://github.com/robotology/gz-yarp-plugins/pull/38#issuecomment-1681909589 for more info
         sudo apt install libgl1 libgl1-mesa-dri libselinux1 libxdamage1 libxxf86vm1 libxext6 libopengl0 libegl1
 

